src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -94,6 +94,7 @@
     private AuthenticationUpdater authenticationUpdater = null;
     private final Set<ConnectionListener> listeners = new CopyOnWriteArraySet<ConnectionListener>();
     private int sshRxBufferSize = SSH_RX_BUFFER_SIZE;
+    private StringBuilder eventBuffer = null;
```

Comment:
I don't see this class intended to be used the way @rinrinne suggests, even though it could potentially be used in that way. And I don't think it _is_ used in that way either. So documenting that it is only intended to be used by itself should suffice in that case if we think that the alternative will suffer big performance penalties.